src/validators/libs.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+
+// TODO: For now we're asuming that all libs are being passed to the watcher
+// We should change @notify-watcher/core to allow the creation of custom execute
+// functions that pass more libs, and then do the validation against those.
+// This also means that @notify-watcher/core should have a way to obtain the list
+// of libs that are being passed to watchers
+
+const fs = require('fs-extra');
+
+const serverLibs = fs.readdirSync('./node_modules');
+
+function validateLibs({ config: { libs }, name }) {
+  const missingLibs = [];
+  for (let i = 0; i < libs.length; i += 1) {
+    const lib = libs[i];
+    if (!serverLibs.includes(lib)) missingLibs.push(lib);
```

Comment:
Sometimes dirs have the name `dir-name`, but the variable they export is imported as `dirName`. Does this work with that? We should probably either change everything to camelCase, or whatever and then do the comparison.

Reply:
What I think we should do is actually change `@notify-watcher/core` to export an `Executor` class.

Then we create an instance of it for executing watchers, this instance will have the preloaded libs (`lodash`, `axios` and `cheerio` for now) but can receive extra libs.

Then `validateLibs` will be a method of the `Executor` object, which checks the list of libs against the one it was provided when created.

This is just for now

(I'm actually working on the `Executor` class now)